BoxContentSDK/BoxContentSDK/BOXURLSessionManager.m

```diff
@@ -57,7 +57,10 @@ - (id)initWithBackgroundSessionId:(NSString *)backgroundSessionId sessionTask:(N
 
 @end
 
-@interface BOXURLSessionManager() <NSURLSessionDelegate, NSURLSessionTaskDelegate, NSURLSessionDataDelegate, NSURLSessionDownloadDelegate, NSURLSessionStreamDelegate>
+@interface BOXURLSessionManager() <NSURLSessionDelegate, NSURLSessionTaskDelegate, NSURLSessionDataDelegate, NSURLSessionDownloadDelegate, NSURLSessionStreamDelegate> {
```

Comment:
Don't need SessionDelegate and SessionTaskDelegate, the others are extensions of those so they are already included.